File: day 78/homework/4.js
let numbers3 = [-8, -2, 0, 3, 6, 9, 11];

let nonNegative = numbers.filter(num => num >= 0);

let added = nonNegative.map(num => num + 5);

let result = added.filter(num => num < 10);

result.forEach(num => {
  console.log(num);
});

File: day 27/homework/4.py
def filter_evens(numbers):
    even_list = []
    
    for num in numbers:
        if num % 2 == 0: # ვამოწმებთ, არის თუ არა ნაშთი 2-ზე გაყოფისას 0
            even_list.append(num)
            
    return even_list

# მაგალითი:
print(filter_evens([1, 2, 3, 4, 5, 6]))  # გამონატანი: [2, 4, 6]